modules/virt-enabling-primary-udn.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc
+
+:_mod-docs-content-type: PROCEDURE                                
+[id="virt-enabling-primary-udn_{context}"]                                  
+= Enabling user-defined networks
+
+To bind a virtual machine (VM) to a user-defined network on the VM's primary interface, you must first set the `primaryUserDefinedNetworkBinding` and `deployKubevirtIpamController` feature gates to `true` in the `HyperConverged` custom resource (CR).
```

Comment:
In 4.18, you no longer need to refer the `deployKubevirtIpamController` FG.

I think you also do not need to do it for the `primaryUserDefinedNetworkBinding` as well. @orelmisan to keep me honest here.